setup.py

```diff
@@ -10,26 +10,26 @@
 except ImportError:
     cythonize = False
 
-# defines whether the python interface should link againt a static (.a)
-# or a shared (.so) scipopt library
-usesharedlib = True
-pathToScipoptsuite = os.path.abspath('../../../')
-if usesharedlib:
-    libscipopt = 'lib/libscipopt.so'
-else:
-    libscipopt = 'lib/libscipopt.a'
-
-linkincludescip = 'lib/scip-src'
+pathToSCIPOpt = os.path.abspath('../../../')
+pathToSCIPsrc = os.path.abspath('../../src')
```

Comment:
Do these relative paths still make sense, now that PySCIPOpt is no longer in the scip repository?